front/temporal/labs/client.ts

```diff
@@ -21,7 +21,7 @@ export async function launchRetrieveTranscriptsWorkflow(
       args: [transcriptsConfiguration.id],
       taskQueue: QUEUE_NAME,
       workflowId: workflowId,
-      cronSchedule: "*/15 * * * *", // Every 15 minutes.
+      cronSchedule: "*/1 * * * *", // Every 15 minutes.
```

Comment:
Have we lost a 5 here?

Reply:
Oups, for my local run... didn't mean to commit